telegraf.js

```diff
@@ -208,11 +208,13 @@ class Telegraf extends Composer {
 
 module.exports = Object.assign(Telegraf, {
   Context,
+  ContextMessageUpdate: Context,
```

Comment:
Im not sure the JavaScript files should export a type which was wrongly introduces by TypeScript?

Reply:
Without this, typings would lie about the class being accessible under this name.

Reply:
The current set of typings are lying all over the place with their current names regarding the JavaScript code and their names in the Telegram API.

This PR should try to fix this situation.
It should only change types and do not interfere with JavaScript. If types have to change the JavaScript then the types have made something very wrong.

If you think this is required to add to module.exports then I prefer not to `export {Context as ContextMessageUpdate}` at all and only export the correct names of things.